src/equations/parabolic_equations.jl

```diff
@@ -0,0 +1,3 @@
+# Linear scalar diffusion for use in linear scalar advection-diffusion problems
```

Comment:
Rename to `equations_parabolic.jl` for consistency?

Reply:
Done - thanks!